src/app/charts/core/files/secrets/tpl/spring-config-saml.properties

```diff
@@ -0,0 +1,45 @@
+{{- $samlconfig := (include "arkcase.saml" $) -}}
+{{- if $samlconfig -}}
+  {{- $conf := get (fromYaml $samlconfig) "saml" -}}
+  {{- if $conf -}}
+  {{- $baseUrl := get (fromYaml $samlconfig) "arkcaseUrl" -}}
```

Comment:
This is one example why using the `fromYaml` once, at the very top, would be beneficial: parse the YAML only once, and then consume the configuration object (i.e. a `dict`) later on, vs. parsing it over and over and over again...